code/Player/Player.cs

```diff
@@ -15,9 +15,14 @@ public partial class Player : Entity
 
 	public PlayerCamera Camera { get; private set; }
 
-	// TODO: Allow the player to choose their own color.
-	[Net]
-	public Color Color { get; private set; } = Color.Random;
+	public Preferences Preferences

```

Comment:
Instead of storing this component in player, let's add it to the Client when they join. That way, when we get around to making the color picker in the menu, the client will already have the preferences component hooked up. This will make less work for us later on.

Then, if we want to be able to access it from the player easily, simply set it up:
`public Preferences Preferences => Client.Components.GetOrCreate<Preferences>();`

Reply:
I misread :)